Update logging for containerization (#5095)
theforeman/foreman-documentation: guides/common/modules/con_managing-logs-for-project.adoc

:_mod-docs-content-type: CONCEPT

[id="managing-logs-for-project"]
= Managing logs for {Project}

[role="_abstract"]

> Centralized logging gives you a single interface to monitor {Project} service health, diagnose errors, and control log retention and disk usage.

Reviewer: This is kind of feature centric, although it does avoid the usual red flags such as "provides".

```suggestion
With centralized logging, you can monitor {Project} service health, diagnose errors, and control log retention and disk usage in a single interface.
```
